changelog.d/6634.bugfix

```diff
@@ -0,0 +1 @@
+Build the same service URL when requesting the CAS ticket and when calling the proxyValidate URL.
```

Comment:
Could you rephrase this in a way that will make more sense to users? Also, feel free to add an attribution here:

```suggestion
Fix single-sign on with CAS systems: pass the same service URL when requesting the CAS ticket and when calling the `proxyValidate` URL. Contributed by @Naugrimm.

```

maybe?